### DISCUSSION TITLE
Perte mot de passe

### DISCUSSION THREAD

**Discussion by aunisien** [FreshRSS/FreshRSS#5157] [Q&A] [answered]
Bonjour, 
depuis plusieurs années j'utilise FreshRSS pour mon plus grand plaisir en auto hébergement avec un seul et unique compte.
Malheureusement j'ai eu la fâcheuse idée de modifier mon mot de passe avec Bitwarden, je n'ai pas sauvegardé dans Bitwarden mais par contre j'ai modifié dans FreshRSS. Y-a-t-il un moyen de contournement pour me connecter et modifier le mot de passe ? 
Merci d'avance pour votre aide

**Comment by Frenzie** [comment, marked as answer]
The easiest would be something like `php cli/update-user.php --user aunisien --password aunisien_password`.

**Reply by aunisien** [reply]
My savior! The request was successfully executed. Thank you very much!
